app/views/calls/create.js.erb

```diff
@@ -1,7 +1,7 @@
-var URL = location.pathname,
-    substring = "edit";
+var isOnEditPage = location.pathname.indexOf('edit') !== 1;
+var isOnDashboard = location.pathname === '/';
```

Comment:
Will swapping out a var for a hardcoded basepath give us any heartburn down the road? I don't think so?

Reply:
I don't _think_ so but the thought did cross my mind. That said our base routes haven't changed in several years at this point.